.ebextensions/07_https-nypl-digital-dev.config

```diff
@@ -0,0 +1,4 @@
+option_settings:
+  aws:elb:loadbalancer:
+    LoadBalancerHTTPSPort: 443
+    SSLCertificateId: arn:aws:acm:us-east-1:946183545209:certificate/d38c5c82-708d-4942-a4cc-bc4985a9a4c7
```

Comment:
Hmm... you're merging to the `development` branch but your SSL ID refers to the production environment (`946183545209`). 

Which brings up a point: should we also create a certificate in the development (`nypl-sandbox`) environment?

Reply:
Yeah, I'm expecting this code to fail, and then I think I'll deactivate it until we can get the stage-sensitive deployment options.

Reply:
I'd like to leave a status report here.  

I pushed this file to development.  I expected development to fail.  Development did not fail.  This is a mystery, but I'll take it.  Looks like the ebextensions file is ignored when there's no load balancer to set the ssl up on.

I'm keeping this file in the code tree, even though it's not actually doing anything on either development or production.

Related note:  Pushing log retention settings in ebextensions also doesn't overwrite the log retention settings made manually in CloudWatch console.